### ISSUE TITLE
Fails 2 tests

### ISSUE THREAD

**Issue by LHMarshall** [DistanceDevelopment/Distance#10]
> devtools::test("Distance")
Loading Distance
Testing Distance
Testing dht : ..
Testing ds() : ....12...........................
Point transects : ......
Testing simplified data tables (tee data) : ......

1. Failure (at test_ds.R#29): Input errors are thrown correctly -----------
ds(egdata, 4, key = "bananas") does not match ''arg' should be one of "hn", "hr", "unif"'. Actual value: "Error in match.arg(key) : 'arg' should be one of “hn”, “hr”, “unif”\n"

2. Failure (at test_ds.R#33): Input errors are thrown correctly -----------
ds(egdata, 4, key = "hn", adjustment = "bananas") does not match ''arg' should be one of "cos", "herm", "poly"'. Actual value: "Error in match.arg(adjustment) : \n  'arg' should be one of “cos”, “herm”, “poly”\n"

**Comment by dill** [comment]
hmmm, I can't reproduce this with what's on github...

Can you try pushing what you have?

(Maybe also need to update your testthat?)

**Comment by LHMarshall** [comment]
Updated testthat, did a pull from github to make sure my files were up-to-date but errors remain.

Also downloaded Distance via .zip file and saved to a new folder and ran devtools::test("Distance") on this and I still got the same errors.

**Comment by LHMarshall** [comment]
hmm maybe I need to update R

**Comment by LHMarshall** [comment]
Nope still got those errors, here is the output when I run those pieces of code outside the test.

> ds(egdata,4,key="bananas")
Error in match.arg(key) : 'arg' should be one of “hn”, “hr”, “unif”
Called from: match.arg(key)
> ds(egdata,4,key="hn",adjustment="bananas")
Error in match.arg(adjustment) : 
  'arg' should be one of “cos”, “herm”, “poly”
Called from: match.arg(adjustment)

**Comment by dill** [comment]
Very strange. I downloaded the .zip of master and got the following, no problem:

```
> devtools::test("Distance")
Loading Distance
Loading required package: testthat
Loading required package: mrds
This is mrds 2.1.15
Built: R 3.2.3; ; 2016-02-06 21:54:12 UTC; unix
Testing Distance
Testing dht : ..
Testing ds() : .................................
Point transects : ......
Testing simplified data tables (tee data) : ......

DONE
```

the output of `sessionInfo()` is:

```
> sessionInfo()
R version 3.2.3 (2015-12-10)
Platform: x86_64-apple-darwin13.4.0 (64-bit)
Running under: OS X 10.10.5 (Yosemite)

locale:
[1] en_GB.UTF-8/en_GB.UTF-8/en_GB.UTF-8/C/en_GB.UTF-8/en_GB.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base

other attached packages:
[1] Distance_0.9.5  mrds_2.1.15     testthat_0.11.0

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.3          magrittr_1.5         roxygen2_5.0.1
 [4] BB_2014.10-1         devtools_1.10.0      svUnit_0.7-12
 [7] lattice_0.20-33      quadprog_1.5-5       Rsolnp_1.16
[10] optimx_2013.8.7      minqa_1.2.4          stringr_1.0.0
[13] optextras_2013-10.28 tools_3.2.3          parallel_3.2.3
[16] grid_3.2.3           dfoptim_2011.8-1     nlme_3.1-122
[19] mgcv_1.8-9           withr_1.0.0          digest_0.6.9
[22] numDeriv_2014.2-1    crayon_1.3.1         ucminf_1.1-3
[25] setRNG_2013.9-1      Matrix_1.2-3         memoise_1.0.0
[28] stringi_1.0-1        truncnorm_1.0-7      Rvmmin_2013-11.12
[31] Rcgmin_2013-2.21

**Comment by LHMarshall** [comment]
Hmm, well I am baffled! Will try on one of the other machines at CREEM when I am in the office on Thurs/Fri (or possibly tomorrow)

```
> sessionInfo()
R version 3.2.3 (2015-12-10)
Platform: x86_64-apple-darwin13.4.0 (64-bit)
Running under: OS X 10.9.5 (Mavericks)

locale:
[1] en_GB.UTF-8/en_GB.UTF-8/en_GB.UTF-8/C/en_GB.UTF-8/en_GB.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] Distance_0.9.5  mrds_2.1.15     testthat_0.11.0

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.3          magrittr_1.5         roxygen2_5.0.1       BB_2014.10-1        
 [5] devtools_1.10.0      svUnit_0.7-12        lattice_0.20-33      quadprog_1.5-5      
 [9] Rsolnp_1.16          optimx_2013.8.7      minqa_1.2.4          stringr_1.0.0       
[13] optextras_2013-10.28 tools_3.2.3          parallel_3.2.3       grid_3.2.3          
[17] dfoptim_2011.8-1     nlme_3.1-124         mgcv_1.8-11          withr_1.0.1         
[21] digest_0.6.9         numDeriv_2014.2-1    crayon_1.3.1         ucminf_1.1-3        
[25] setRNG_2013.9-1      Matrix_1.2-3         memoise_1.0.0        stringi_1.0-1       
[29] truncnorm_1.0-7      Rvmmin_2013-11.12    Rcgmin_2013-2.21   

DONE
```

**Comment by dill** [comment]
This is fixed in 4dcfdef. But [@thosjleeper recommends](https://twitter.com/thosjleeper/status/763760361325662209) using the regex `[]` set operators to put in all the quotes we need.

This seems a bit less voodoo-like than the current solution, but I don't think this is high priority at all.
